Dockerfile

```diff
@@ -29,22 +26,32 @@ RUN $PIP_INSTALL --user --upgrade pip pillow oauthlib pyjwt httplib2 numpy
 
 RUN $APT_UPDATE && $APT_INSTALL \
     unzip software-properties-common apt-transport-https \
-    openjdk-11-jdk xvfb siege apache2-utils firefox ruby ruby-dev make nodejs locales tsung
+    openjdk-11-jdk xvfb siege apache2-utils ruby ruby-dev make nodejs locales tsung
+
+# Fix vulnerabilities / outdated versions
+RUN $PIP_INSTALL --user --upgrade pip pillow oauthlib pyjwt httplib2 numpy
+
+RUN $APT_UPDATE && $APT_INSTALL \
```

Comment:
why are we running this twice?

RUN $APT_UPDATE && $APT_INSTALL \
    unzip software-properties-common apt-transport-https \
    openjdk-11-jdk xvfb siege apache2-utils ruby ruby-dev make nodejs locales tsung

Reply:
wrong merge, will remove it